internal/driver/driver.go

```diff
@@ -90,7 +90,12 @@ func (linodeDriver *LinodeDriver) SetupLinodeDriver(
 	// Set up RPC Servers
 	linodeDriver.ids = NewIdentityServer(linodeDriver)
 	linodeDriver.ns = NewNodeServer(linodeDriver, mounter, deviceUtils, linodeClient, metadata)
-	linodeDriver.cs = NewControllerServer(linodeDriver, linodeClient, metadata)
+
+	cs, err := NewControllerServer(linodeDriver, linodeClient, metadata)
+	if err != nil {
+		return fmt.Errorf("new controller server: %w", err)
+	}
+	linodeDriver.cs = cs
```

Comment:
not a hill i'll die on but I know you're a go syntax nerd... would you consider this change? not sure the wrapped error adds too much and it's a bit simpler, did this blind so maybe double check it's valid

```suggestion
	linodeDriver.cs, err = NewControllerServer(linodeDriver, linodeClient, metadata)
	return err
```

Reply:
That's equally as valid! Though, I would still want to wrap the error, simply so that _if it does fail_, the error wrappings act like a breadcrumb trail to trace the error.

Reply:
ya of course, leave it if you like it. nothing wrong with more verbosity